transport/grpc/codec.go

```diff
@@ -0,0 +1,29 @@
+package grpc
+
+import "fmt"
+
+// PassThroughCodec passes bytes to/from the wire without modification
+type PassThroughCodec struct{}
```

Comment:
+1 to unexport. We don't want this to be part of the public API.  Or if you
really don't want to copy paste this, place it in `internal/transport/grpc`.
`examples/` will still have access to it but not packages outside yarpc.